route/src/route.cpp

```diff
@@ -32,7 +32,7 @@ namespace route {
         route_state_pub_ = nh_->advertise<cav_msgs::RouteState>("route_state", 1, true);
         route_marker_pub_= nh_->advertise<visualization_msgs::Marker>("route_marker", 1, true);
         // init subscribers
-        pose_sub_ = nh_->subscribe("current_pose", 1, &RouteGeneratorWorker::pose_cb, &rg_worker_);
+        // pose_sub_ = nh_->subscribe("current_pose", 1, &RouteGeneratorWorker::pose_cb, &rg_worker_);
```

Comment:
If these are no longer needed due to the front_bumper transform, let's delete these lines entirely.

Reply:
removed